lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -443,6 +443,12 @@ function dosomething_reportback_view_entity($entity) {
     $subtitle = ($is_owner) ? $copy_vars['owners_rb_subtitle'] . " " . $incentive : $node->call_to_action;
     $link = '/' . drupal_get_path_alias('node/' . $node->nid);
 
+    // Share button links
+    //@TODO - Pass the correct link in.
+    $fb_link = paraneue_dosomething_facebook_intent("http://www.dosomething.org");
+    $twitter_link = paraneue_dosomething_twitter_intent("http://www.dosomething.org");
+    $tumblr_link = paraneue_dosomething_tumblr_intent("http://www.dosomething.org");
```

Comment:
Just a thought, but could be refactored so that the default value for `$link` is `http://www.dosomething.org` instead of having to pass it each time, and only pass a link if you want to override the default.